packages/reakit/src/Hidden/index.ts

```diff
@@ -4,9 +4,13 @@ import HiddenHide from "./HiddenHide";
 import HiddenShow from "./HiddenShow";
 import HiddenToggle from "./HiddenToggle";
 
+// @ts-ignore
 Hidden.Container = HiddenContainer;
+// @ts-ignore
 Hidden.Hide = HiddenHide;
+// @ts-ignore
 Hidden.Show = HiddenShow;
+// @ts-ignore
 Hidden.Toggle = HiddenToggle;
```

Comment:
What do you think about this?
https://github.com/reakit/reakit/blob/62b108cd9f9656512ac1bda3d166fa33196de955/packages/reakit/src/Card/index.ts#L1-L12

Reply:
Interesting, definitely better then `@ts-ignore`